PORTAL/jobs/_utils.py

```diff
@@ -341,7 +415,11 @@ def render_count(self, total=None, label='matched'):
 
 class TableRow:
 
-    def __init__(self, data, render_value):
+    def __init__(
+            self,
+            data: Mapping[str, Any],
```

Comment:
It's seems odd that the value type here would be tricky to define.

Reply:
According to its one usage in `_jobs.py:Job.as_row` it's:

```
Union[RequestID, str, None, Tuple[Optional[datetime], Optional[datetime]], datetime, timedelta, Any]
```

When things get that generic, I've just been calling it `Any`.  I think strictly speaking, it's probably "anything with a `__str__`".

I'll leave this as `Any` for now, unless you see a good reason to be more specific.

Reply:
Ah, right.  I was thinking of it as rendered data or something.  This is before the values get rendered.